test/visual/accordion.html

```diff
@@ -1,20 +1,8 @@
 <!DOCTYPE html>
-
+<html></html>
```

Comment:
This is not right. 😄 Should only have the beginning tag here.

Reply:
Thanks, my bad 😄 Fixed.